physionet-django/templates/base.html

```diff
@@ -8,6 +8,9 @@
     {% include "base_css.html" %}
     {% block local_css %}{% endblock %}
     {% include "base_js_top.html" %}
+    {% if SESSION_SECURITY_ENABLED %}
+      {% include 'session_security/all.html' %}
+    {% endif %}
```

Comment:
Maybe move this one line below. Then it's ordered a bit nicer:
1. CSS
2. JS
3. Session